src/main/java/picard/vcf/VcfToIntervalList.java

```diff
@@ -84,7 +84,7 @@
     @Argument(doc="Controls the naming of the resulting intervals. When true, each resulting interval will be named the concatenation of " +
             "the variant ID fields (if present), or 'interval-<number>' (if not) with a pipe '|' separator. " +
             "When false, only the first name will be used.")
-    public boolean CONCATENATE_IDS = true;
+    public boolean CONCATENATE_IDS = false;
```

Comment:
Hmmm. The reason that this was default `true` was that that was the behavior prior to making this an option....we didn't want to change the default behavior....why does the default need to change?

Reply:
I thought boolean arguments are supposed to default to false? I could make it 
 `DONT_ CONCATENATE_IDS` or something or maybe it doesn't matter enough

Reply:
yeah, make the default value `false`, but keep the default behavior the same...or define an enum "CONCAT_ALL" & "USE_FIRST" to control the behaviour (with the default value being CONCAT_ALL)

Reply:
Yeah, I like the enum idea better than a double negative (too confusing) 👍